packages/big-design/src/components/Form/Label/styled.tsx

```diff
@@ -19,12 +19,12 @@ export const StyledLabel = styled<
   display: inline-block;
   margin-bottom: ${({ theme }: StyledLabelArgument) => theme.spacing.xxSmall};
 
-  ${({ theme, renderOptional }: StyledLabelArgument) =>
+  ${({ theme, renderOptional, optionalLabel }: StyledLabelArgument) =>
     renderOptional &&
     css`
       &::after {
         color: ${theme.colors.secondary60};
-        content: ' (optional)';
+        content: ' (${optionalLabel?.toLowerCase() ?? 'optional'})';
```

Comment:
we shouldn't be enforcing this, it is up to the consumer to decide if they want lowercase or uppercase.